### ISSUE TITLE
Can't run with a static vm with a disk unless you specify disk_size

### ISSUE THREAD

**Issue by ehankland** [GoogleCloudPlatform/PerfKitBenchmarker#945]
I got this error:
```
2016-04-12 10:07:11,731 ee17f302 MainThread fio(1/1) ERROR    Error during benchmark fio
Traceback (most recent call last):
  File "/usr/local/google/home/ehankland/PerfKitBenchmarker/perfkitbenchmarker/pkb.py", line 381, in RunBenchmark
    detailed_timer)
  File "/usr/local/google/home/ehankland/PerfKitBenchmarker/perfkitbenchmarker/pkb.py", line 269, in DoRunPhase
    collector.AddSamples(samples, name, spec)
  File "/usr/local/google/home/ehankland/PerfKitBenchmarker/perfkitbenchmarker/publisher.py", line 598, in AddSamples
    sample['metadata'], benchmark_spec)
  File "/usr/local/google/home/ehankland/PerfKitBenchmarker/perfkitbenchmarker/publisher.py", line 168, in AddMetadata
    data_disk.disk_size * data_disk.num_striped_disks)
TypeError: unsupported operand type(s) for *: 'NoneType' and 'int'
```
Looks like the problem is this [line](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/publisher.py#L168)

**Comment by ehankland** [comment]
@noahl This looks related to your disk refactor - mind taking a look?

**Comment by noahl** [comment]
@ehankland yep, I've got it. Looks like with the old metadata keys, we just ignore or use None as the size if we don't know it, but the computation assumes that we actually have that information.

**Comment by noahl** [comment]
This should be fixed with the merge of #946 . Closing now, but please reopen if you see more problems.
